pages/lyrics/[id].vue

```diff
@@ -0,0 +1,221 @@
+<script setup lang="ts">
+import { ContributorApi, LyricsApi } from "@bcc-code/bmm-sdk-fetch";
+import type { ContributorModel, Lyrics } from "@bcc-code/bmm-sdk-fetch";
+
+const route = useRoute("lyrics-id");
+
+const lyrics = ref<Lyrics>();
+
+onMounted(async () => {
+  lyrics.value = await new LyricsApi().lyricsIdGet({
+    id: Number(route.params.id),
+  });
+
+  // Default longCopyright if not set
+  lyrics.value.longCopyright ??= `© Stiftelsen Skjulte Skatters Forlag, Norway. All rights reserved.
+forlaget@skjulte-skatter.no / activechristianity.org / christianbookshop.org`;
```

Comment:
I think we should only have the copyright information. Not the promotion of some websites.